### PR TITLE
Completed listing form, added post request functionality

### PR DESCRIPTION
**Pull request by arturob1024**

### CODE REVIEW

**Comment by arturob1024** [comment]
I renamed listingPage to listingForm because it's a component not a page. When the merge occurs, git might not remove listingPage. Have to remove in future commit.

File: controllers/listings.js [hunk 1]
```diff
@@ -13,4 +13,18 @@ exports.getListings = (req,res) => {
             res.json(listing); // return all listings in JSON format
         })
 
-};
\ No newline at end of file
+};
+
+exports.addListing = (req,res) => {
+    console.log("bazinga");
```
Comment (LamLieu) [inline controllers/listings.js:19]: Change console log message so the others would know that a listing was added.

File: client/debug.log [hunk 2]
```diff
@@ -1 +1,3 @@
 [1026/181233.369:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[1107/102443.496:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
+[1107/124554.112:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (LamLieu) [inline client/debug.log:3]: Add this file to .gitignore file

**Review by LamLieu — CHANGES_REQUESTED** [review]
Overall looks good, just a few fixes/additions.

**Comment by LamLieu** [comment]
> I renamed listingPage to listingForm because it's a component not a page. When the merge occurs, git might not remove listingPage. Have to remove in future commit.

If you look at the "Files changed" tab, you will see that "FormPage" was removed and "listingForm" was added. The merge will be ok! 👍

**Review by LamLieu — APPROVED** [review]

**Pull request merged**